### PR TITLE
Added Russian translation

### PR DESCRIPTION
**Pull request by mimizid**
**What type of PR is this.**
 

- [ ] Bug
- [x] Enhancement

**What have you changed and why?**
Add Russian localization

**Please specify which Issue this PR Resolves.**
NONE

**Please verify the following and ensure all checks are completed.**
 
- [x] Have you loaded the Mission in Singleplayer?
- [x] Have you loaded the Mission in a Dedicated Server?

**Is further testing or are further changes required?**

- [x] No
- [ ] Yes (Please provide further detail below.)

### CODE REVIEW

**Comment by Bob-Murphy** [comment]
Hi mimizid,
thank you very much for your PRs!
As we currently have a lot of stuff in the pipeline it can take some time to get everything properly reviewed.
For the russian translations, we also will let somebody with russian capabilities check your translations as we have a general 4-eye-principle.
We will certainly use your changes even though we might ask for slight adaptations.
Nevertheless, well done and thank you.

**Comment by Bob-Murphy** [comment]
Format is reviewed
Still awaiting review for russian translation

File: A3-Antistasi/Stringtable.xml [hunk 1]
```diff
@@ -1105,10 +1118,12 @@
       <Key ID="STR_antistasi_mission_info_chernawinter_mapname_short_text">
         <Original>Antistasi Chernarus Winter</Original>
         <English>Antistasi Chernarus Winter</English>
+        <Russian>Antistasi Чернорусь Зима</Russian>
       </Key>
       <Key ID="STR_antistasi_mission_info_chernawinter_mapname_text">
         <Original>Antistasi - Chilled revolt</Original>
         <English>Antistasi - Chilled revolt</English>
+        <Russian>Antistasi - Революция колхоза</Russian>
```
Comment (AlexTriada) [inline A3-Antistasi/Stringtable.xml:1126]: Antistasi - колхозная Революция

File: A3-Antistasi/Stringtable.xml [hunk 2]
```diff
@@ -1335,6 +1359,7 @@
         <Czech>Začátek</Czech>
         <Chinesesimp>早期开始</Chinesesimp>
         <Korean>초기 시작</Korean>
+        <Russian>Раннее начало</Russian>
```
Comment (AlexTriada) [inline A3-Antistasi/Stringtable.xml:1362]: В самом начале

File: A3-Antistasi/Stringtable.xml [hunk 3]
```diff
@@ -1556,6 +1605,7 @@
         <Czech>Bombardování</Czech>
         <Chinesesimp>空中打击</Chinesesimp>
         <Korean>폭격</Korean>
+        <Russian>Авиоудар</Russian>
```
Comment (AlexTriada) [inline A3-Antistasi/Stringtable.xml:1608]: Авиаудар

File: A3-Antistasi/Stringtable.xml [hunk 4]
```diff
@@ -1716,6 +1785,7 @@
         <Czech>Vrchní velení</Czech>
         <Chinesesimp>高级指挥</Chinesesimp>
         <Korean>High Command</Korean>
+        <Russian>Высшее командование</Russian>
```
Comment (AlexTriada) [inline A3-Antistasi/Stringtable.xml:1788]: Командование отрядами

File: A3-Antistasi/Stringtable.xml [hunk 5]
```diff
@@ -1860,6 +1947,7 @@
         <Czech>Rádiové věže</Czech>
         <Chinesesimp>无线电发射塔</Chinesesimp>
         <Korean>라디오 타워</Korean>
+        <Russian>Радио вышки</Russian>
```
Comment (AlexTriada) [inline A3-Antistasi/Stringtable.xml:1950]: Радиовышки

File: A3-Antistasi/Stringtable.xml [hunk 6]
```diff
@@ -1924,6 +2019,7 @@
         <Czech>Přenos ammuboxu</Czech>
         <Chinesesimp>弹药箱转移</Chinesesimp>
         <Korean>탄약박스 운반</Korean>
+        <Russian>Передача боеприпасов</Russian>
```
Comment (AlexTriada) [inline A3-Antistasi/Stringtable.xml:2022]: Погрузка боеприпасов

File: A3-Antistasi/Stringtable.xml [hunk 7]
```diff
@@ -2060,6 +2172,7 @@
         <Czech>Na některých frakčních vlajkách a na velitelských vlajkách můžete najmout až 9 jednotek AI.</Czech>
         <Chinesesimp>你可以在一些派系旗和总部旗招募最多9名AI队友.</Chinesesimp>
         <Korean>HQ 깃발과 일부 팩션 깃발에서 최대 9명의 AI 분대원을 모집할 수 있습니다.</Korean>
+        <Russian>Вы можете нанять до 9 товарищей по отряду ИИ.</Russian>
```
Comment (AlexTriada) [inline A3-Antistasi/Stringtable.xml:2175]: Вы можете нанять до 9 ИИ товарищей в отряд

File: A3-Antistasi/Stringtable.xml [hunk 8]
```diff
@@ -2084,6 +2199,7 @@
         <Czech>AI Rádio</Czech>
         <Chinesesimp>AI无线电</Chinesesimp>
         <Korean>AI 라디오</Korean>
+        <Russian>Радио ИИ</Russian>
```
Comment (AlexTriada) [inline A3-Antistasi/Stringtable.xml:2202]: Радиосвязь с ИИ

File: A3-Antistasi/Stringtable.xml [hunk 9]
```diff
@@ -2116,6 +2235,7 @@
         <Czech>Automatické přezbrojení \ Rabování</Czech>
         <Chinesesimp>自动 武装 \ 掠夺</Chinesesimp>
         <Korean>자동획득 \ 약탈</Korean>
+        <Russian>Автопополнение \ Лут</Russian>
```
Comment (AlexTriada) [inline A3-Antistasi/Stringtable.xml:2238]: Авто-снаряжение

**Comment by AlexTriada** [comment]
In General, very good!

**Comment by Bob-Murphy** [comment]
@mimizid 
Please check the comments made by Alex and resolve them.
Thank you.

File: A3-Antistasi/Stringtable.xml (hunk 3 above)
Comment (mimizid) [reply A3-Antistasi/Stringtable.xml:1608]: Facepalm

**Comment by AlexTriada** [comment]
translate is good

**Comment by Bob-Murphy** [comment]
Thank you. Will look at it on Monday.

**Review by Bob-Murphy — APPROVED** [review]

**Pull request merged**